### PR TITLE
Navigation new entry

### PR DESCRIPTION
**Pull request by SergeBrakow**
Created pages (new, order, customer) and navigation between them.
Outsourced the Navbar element.

### CODE REVIEW

File: src/components/NavBarNewOrder.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+export const StyledNavBar = styled.section`
+  border-top: 1px solid;
+  background-color: white;
+  display: flex;
+  justify-content: space-between;
+  position: fixed;
+  width: 100%;
+  height: 50px;
+  bottom: 0;
+  layer: 10;
```
Comment (ThomasF85) [inline src/components/NavBarNewOrder.js:45]: Invalid css

File: src/components/NavBarNewOrder.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        localStorage.removeItem(`user`);
+        navigate("/login");
+    }
+    
+    return(
+        <StyledNavBar>
+        {prop.page === "home" ?
+                <>
+                <NavBarElement>
+                <button onClick={()=> logout()}>{prop.user} logout</button>
```
Comment (ThomasF85) [inline src/components/NavBarNewOrder.js:18]: No need to create empty wrappers for functions

**Review by ThomasF85 — CHANGES_REQUESTED** [review]
Just some small changes requested. Please take a look at the comments.

File: src/components/NavBarNewOrder.js (hunk 1 above)
Comment (SergeBrakow) [reply src/components/NavBarNewOrder.js:45]: thx.

File: src/components/NavBarNewOrder.js (hunk 2 above)
Comment (SergeBrakow) [reply src/components/NavBarNewOrder.js:18]: If i leave it out an error is displayed.
If i replace it with a div, the style is inherited on it. What I not want to.

Comment (SergeBrakow) [reply src/components/NavBarNewOrder.js:18]: done

**Review by jecaps — APPROVED** [review]

**Pull request merged**